i18n/en.json

```diff
@@ -1528,10 +1528,6 @@
     "id": "api.plugin.install.reading_stream_failed.app_error",
     "translation": "An error ocurred reading the plugin file stream."
   },
-  {
-    "id": "api.plugin.install.verify_plugin.app_error",
```

Comment:
dont' we want to return this if verification fails when installing a marketplace plugin?

Reply:
We do have it 
https://github.com/mattermost/mattermost-server/blob/93d062b2da3c63362c1fed7a97db2f4a969948c9/app/plugin_signature.go#L97

Reply:
oh okay a different type of error, cool 👌